core/common/src/test/java/alluxio/ConfigurationTest.java

```diff
@@ -507,4 +507,22 @@ public void sitePropertiesLoadedNotInTest() throws Exception {
       assertEquals("TEST_LOGGER", Configuration.get(PropertyKey.LOGGER_TYPE));
     }
   }
+
+  @Test
+  public void discardIgnoredSitePropertiesTest() throws Exception {
```

Comment:
We don't append "Test' to our unit tests

Reply:
Done.